db/migrate/20160901010430_change_column_type.rb

```diff
@@ -1,13 +0,0 @@
-class ChangeColumnType < ActiveRecord::Migration
```

Comment:
I removed this because there were two migration files with the exact same thing.  The only difference is that one had a change method and the one I removed didn't.